qiskit/circuit/library/standard_gates/i.py

```diff
@@ -52,7 +52,15 @@ def __init__(self, label: Optional[str] = None, *, duration=None, unit="dt"):
     _singleton_lookup_key = stdlib_singleton_key()
 
     def inverse(self, annotated: bool = False):
-        """Invert this gate."""
+        """Returne the inverse gate (itself).
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this gate is self inverse.
```